models/public/mozilla-deepspeech-0.8.2/pbmm_to_pb.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+#
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import argparse
+
+try:
+    from tensorflow import GraphDef
+except ImportError:
+    from tensorflow.compat.v1 import GraphDef
+
+from mds_convert_utils.memmapped_file_system_pb2 import MemmappedFileSystemDirectory
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Convert memory-mapped Tensorflow protobuf format into plain Tensorflow protobuf (GraphDef)")
+    parser.add_argument('input', type=str, help="Input .pbmm filename")
+    parser.add_argument('output', type=str, help="Output .pb filename")
```

Comment:
You should be using `Path` as the type.

Reply:
Done in both scripts, including scorer_to_kenlm.py.